pandasai/constants.py

```diff
@@ -8,8 +8,9 @@
 START_CODE_TAG = "<startCode>"
 END_CODE_TAG = "<endCode>"
 WHITELISTED_LIBRARIES = [
+    "pandas",
```

Comment:
Prior to https://github.com/gventuri/pandas-ai/commit/495f4c441ecfdd09982436994dc3dbc2ac2cfe92, the behaviour was that imports of `pandas` were removed from the run code. The logic as is would keep them in if added to WHITELISTED_LIBRARIES. Is that what we want?

Reply:
yes, i remember but now we are raising the import error before generating the code atleast pandas can be on whitelist. maybe @gventuri can confirm?

Reply:
yes I am aware but how do we surpass this since now we are throwing bad import error before generating & exec the code.

Reply:
Currently when `_is_unsafe_import()` returns `True`, the unsafe library is removed from the generated code without raising an exception. When the `pandas` is imported, `_is_unsafe_import()` should return `True`. One possible implementation:
https://github.com/gventuri/pandas-ai/blob/55178f873d3b3db6e98d1726a42c7f5b8aa89bc8/pandasai/__init__.py#L321
```python
if alias.name in WHITELISTED_BUILTINS or alias.name == "pandas":
    return True
```